src/controller/CommissioningWindowOpener.cpp

```diff
@@ -183,15 +211,34 @@ CHIP_ERROR CommissioningWindowOpener::OpenCommissioningWindowInternal(Messaging:
         MutableByteSpan serializedVerifierSpan(serializedVerifier);
         ReturnErrorOnFailure(mVerifier.Serialize(serializedVerifierSpan));
 
-        AdministratorCommissioning::Commands::OpenCommissioningWindow::Type request;
-        request.commissioningTimeout = mCommissioningWindowTimeout.count();
-        request.PAKEPasscodeVerifier = serializedVerifierSpan;
-        request.discriminator        = mDiscriminator.GetLongValue();
-        request.iterations           = mPBKDFIterations;
-        request.salt                 = mPBKDFSalt;
-
-        ReturnErrorOnFailure(cluster.InvokeCommand(request, this, OnOpenCommissioningWindowSuccess,
-                                                   OnOpenCommissioningWindowFailure, MakeOptional(kTimedInvokeTimeoutMs)));
+#if CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+        if (mJointCommissioning == false)
+        {
+            AdministratorCommissioning::Commands::OpenCommissioningWindow::Type request;
+            request.commissioningTimeout = mCommissioningWindowTimeout.count();
+            request.PAKEPasscodeVerifier = serializedVerifierSpan;
+            request.discriminator        = mDiscriminator.GetLongValue();
+            request.iterations           = mPBKDFIterations;
+            request.salt                 = mPBKDFSalt;
+
+            ReturnErrorOnFailure(cluster.InvokeCommand(request, this, OnOpenCommissioningWindowSuccess,
+                                                       OnOpenCommissioningWindowFailure, MakeOptional(kTimedInvokeTimeoutMs)));
+        }
+        else
+        {
+#endif // CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC
+            JointFabricAdministrator::Commands::OpenJointCommissioningWindow::Type request;
```

Comment:
This looks broken.  If CHIP_DEVICE_CONFIG_ENABLE_JOINT_FABRIC is false, it will try to OpenJointCommissioningWindow, no?

Looks like the bot caught this too.

Reply:
Yes, addressed this in the latest commit.